Forescoop/Classes/ForecastWindguruMockup.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  ForecastWindguruMockup.swift
+//  Forescoop
+//
+//  Created by fox on 07/06/2023.
+//
+
+import Foundation
```

Comment:
here the mockup, need to implement this to use the json files that are already in the unit tests.